### PR TITLE
Ankit | test | Prod - Invalid date error is coming when changing the entry date on ledger for imported transaction 

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cw4245t

### CODE REVIEW

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts [hunk 1]
```diff
@@ -55,6 +55,8 @@ export class GiddhDatepickerComponent implements ControlValueAccessor, OnInit, O
     @Output() public focusOut: EventEmitter<any> = new EventEmitter<any>();
     /** This will hold if datepicker is open */
     public isDatepickerOpen: boolean = false;
+     /** This is used to show change date */
+     public inputChange : any ='';
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts]:
```suggestion
     public inputChange: any = '';
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts [hunk 2]
```diff
@@ -96,14 +98,24 @@ export class GiddhDatepickerComponent implements ControlValueAccessor, OnInit, O
         this.destroyed$.complete();
     }
     
+    /**
+     * get current value on input 
+     * 
+     * @param event 
+     */
+    public dateInputChange(event: Event): void {
+        const inputElement = event.target as HTMLInputElement;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts:108]:
```suggestion
if (event) {
        const inputElement = event.target as HTMLInputElement;
```

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts [hunk 3]
```diff
@@ -96,14 +98,24 @@ export class GiddhDatepickerComponent implements ControlValueAccessor, OnInit, O
...
+        const inputValue = inputElement.value;
+        this.inputChange = inputValue;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts:110]:
```suggestion
        this.inputChange = inputValue;
      }
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts:110]: done

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts:108]: done

File: apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/theme/giddh-datepicker/giddh-datepicker.component.ts]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**